File: Python/count_digits.py
def main():
    n = int(input("Enter a number: "))
    n = abs(n)

    if n == 0:
        print("Count of digits:", 1)
        return

    count = 0
    while n > 0:
        count += 1
        n //= 10

    print("Count of digits:", count)


if __name__ == "__main__":
    main()

File: C/count_digits.c
#include <stdio.h>
#include <stdlib.h>

int main() {
    long long n;
    printf("Enter a number: ");
    if (scanf("%lld", &n) != 1) return 0;

    long long x = llabs(n);
    int count = 0;

    if (x == 0) {
        count = 1;
    } else {
        while (x > 0) {
            count++;
            x /= 10;
        }
    }

    printf("Count of digits: %d\n", count);
    return 0;
}

